Source/WebCore/editing/TextIterator.cpp

```diff
@@ -586,7 +587,7 @@ bool TextIterator::handleTextNode()
         if (renderer.style().visibility() != Visibility::Visible && !m_behaviors.contains(TextIteratorBehavior::IgnoresStyleVisibility))
             return false;
         int rendererTextLength = rendererText.length();
-        int end = (&textNode == m_endContainer) ? m_endOffset : INT_MAX;
+        int end = (textNode.ptr() == m_endContainer) ? m_endOffset : INT_MAX;
```

Comment:
I wonder if we should overload `==` on `Ref` so we can do these without writing `ptr()`?